packages/@react-aria/aria-modal-polyfill/src/index.ts

```diff
@@ -34,14 +34,15 @@ export function watchModals(selector:string = 'body', {document = currentDocumen
   let undo: Revert | undefined;
 
   let observer = new MutationObserver((mutationRecord) => {
+    const liveAnnouncer: HTMLElement =  document.querySelector('[data-live-announcer="true"]');
```

Comment:
so this selects the top level one and assumes only one per application. this should probably be more general though. if we did
```
const liveAnnouncers: HTMLElement = document.querySelectorAll('[aria-live]');
///
undo = hideOthers([modal, ...liveAnnouncers])
```

that would be useful in other applications as well or with other libraries that implement their own live announcers

Reply:
I don't think we necessarily want to preserve every `aria-live` element though, just our live announcer.